### PR TITLE
Script for coverage when RPC with autoCompleteList not allow

### PR DESCRIPTION
**Pull request by Aleksandriys**
@aarotar @OVikhrov please review.

### CODE REVIEW

File: test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua [hunk 1]
```diff
@@ -0,0 +1,185 @@
+-----------------------------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "DISALLOWED"> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua]: @Aleksandriys, please update summary  - currently SDL is used twice: "..SDL SDL ..."

File: test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+--
+-- Description:
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua]: @Aleksandriys,   
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. please update description with particular resultCode from HMI (please specify "DISALLOWED") and not <resultCode_received_from _HMI>

File: test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+-- SDL doesn’t transfer not allowed "SetGlobalProperties"_request with valid ""autoCompleteList" param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList>.
+-- 3. SDL respond <success = true, resultCode = "DISALLOWED"> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua]: @Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3." SDL respond <success = true, resultCode = "DISALLOWED"> to mobile app" Should be in expected result.

File: test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+local mobile_session = require('mobile_session')
+require('user_modules/AppTypes')
+
+--[[ Local Functions ]]
+local registerAppInterfaceParams =
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua]: @Aleksandriys, **local registerAppInterfaceParams** should be specified under --[[ Local variables ]]

File: test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+function Test.Precondition_Backup_preloadedPT()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+end
+
+function Test.DisallowSetGlobalProperties()
+  UpdatePreloadedJson_DisallowSetGlobalPropertiesRPC()
+end
+
+--[[Test]]
+commonFunctions:newTestCasesGroup("Test")
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua]: @Aleksandriys, functions:
_StartSDL_With_DISALLOWED_SetGlobalProperties()
_initHMI()
_initHMI_onReady()
_ConnectMobile()
_CreateSession()
_RegisterApplication()
could be moved to preconditions also.

**Comment by Aleksandriys** [comment]
@aarotar please review.

**Pull request merged**